package.json

```diff
@@ -25,11 +25,13 @@
     "@types/jest": "^24.0.20",
     "@types/node": "^11.10.4",
     "@types/request-promise-native": "^1.0.17",
+    "@types/sinon": "^9.0.0",
     "jest": "^24.9.0",
     "prettier": "^1.19.1",
     "serverless": "^1.38.0",
     "serverless-iam-roles-per-function": "^2.0.2",
     "serverless-webpack": "^5.2.0",
+    "sinon": "^9.0.2",
```

Comment:
If at all possible, I'd like to stick with jest mocking instead of bringing in another framework.

I found some notes on a transition [here](https://github.com/skovhus/jest-codemods/issues/68#issuecomment-363054214).

Reply:
makes sense. Thanks for pointing this out. I thought jest was limited to test execution and didn't know it contained mock functionality. Thanks!

Reply:
fixed in https://github.com/troyready/serverless-artifact-cacher/pull/3/commits/ee0cb6bfd24076a9b7c5ad6fcf4c7998d32825d6.

Thanks for the link with the instructions. it was very helpful. thanks!